extensions/amp-fit-text/0.2/fit-text.css.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export const LINE_HEIGHT_EM_ = 1.15;
+
+export const fitTextContent = {
+  'contain': 'content',
+  'display': 'flex',
```

Comment:
I'm debating whether we should just leave flexbox to the AMP level only as well. This is pretty subjective and hard to override. WDYT?

Reply:
IMO the component is about filling available space of a specified area, and the vertical centering afforded by flex indicates a stronger relationship between the content given and the space its given. 

![Screen Shot 2020-05-08 at 10 34 32 AM](https://user-images.githubusercontent.com/10456171/81417232-db237c80-9118-11ea-8eb5-1cfd7e818fa6.png)
![Screen Shot 2020-05-08 at 10 44 03 AM](https://user-images.githubusercontent.com/10456171/81417237-dced4000-9118-11ea-87e9-6d1ad7c63d57.png)

Reply:
Agree.